README.md

```diff
@@ -482,7 +482,7 @@ Commands:
 
 To run Cromwell in a notebook instance:
 
-* Run `terra cromwell generate-config [--dir=my/path]`
+* Run `terra cromwell generate-config (--workspace-bucket-name=bucket_name | --google-bucket-name=my-bucket) [--dir=my/path]`
```

Comment:
i still would like to keep the gs:// part. I think that's pretty clear to someone who knows gcs bucket what it is based on that hint.

Reply:
Fixed.